openquake/hazardlib/tests/gsim/abrahamson_2018_test.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2015-2018 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+from openquake.hazardlib.gsim.abrahamson_2018 import (
+    AbrahamsonEtAl2018SInter,
+    AbrahamsonEtAl2018SInterHigh,
+    AbrahamsonEtAl2018SInterLow,
+    AbrahamsonEtAl2018SSlab,
+    AbrahamsonEtAl2018SSlabHigh,
+    AbrahamsonEtAl2018SSlabLow)
+from openquake.hazardlib.tests.gsim.utils import BaseGSIMTestCase
+
+
+# Interface
+class AbrahamsonEtAl2018SInterTestCase(BaseGSIMTestCase):
```

Comment:
Can you pls add a note specifying how you created the verification tables? Thanks